Using REST APIs with ADAL (ADAL C# Sample)/README.md

```diff
@@ -0,0 +1,33 @@
+# Using REST APIs with Adal (Adal C# Sample)
+
+The Azure Active Directory Authentication Library (ADAL) enables client application developers to authenticate cloud users and obtain access tokens for API usage.
+
+## Sample Application
+
+This buildable sample will walk you through the steps to create a native console application which uses ADAL to authenticate a user and return a list of all projects inside a selected VSTS account.
```

Comment:
Native has multiple meanings so we need to be careful here.  I think should remove the term as native console sounds like it's a C++ app which this is not, it's managed :)  The intention of "native" was that the application is client side so let's use that instead in all locations